contracts/src/challengeV2/libraries/EdgeChallengeManagerLib.sol

```diff
@@ -421,6 +424,15 @@ library EdgeChallengeManagerLib {
         (bytes32 startHistoryRoot) = layerZeroCommonChecks(proofData, args, expectedEndHeight);
         // we only wrap the struct creation in a function as doing so with exceeds the stack limit
         ChallengeEdge memory ce = toLayerZeroEdge(originId, startHistoryRoot, args);
+
+        if (whitelistEnabled) {
```

Comment:
add some comment on why we are doing this when `whitelistEnabled`
or rename `whitelistEnabled` into something more self explanatory